packages/vuetify/src/components/VSnackbar/VSnackbar.ts

```diff
@@ -171,6 +172,13 @@ export default mixins(
           name: 'show',
           value: this.isActive,
         }],
+        on: this.holdOnHover ? {
+          mouseenter: () => {
+            window.clearTimeout(this.activeTimeout)
+            this.activeTimeout = -1
```

Comment:
Doesn't need to be reset, timeout numbers aren't reused and clearing the same one twice doesn't do anything.